src/functionalTest/groovy/com/autonomousapps/jvm/projects/ClasspathConfusionProject.groovy

```diff
@@ -81,7 +79,10 @@ final class ClasspathConfusionProject extends AbstractProject {
     return actualProjectAdvice(gradleProject)
   }
 
-  private final Set<Advice> consumerAdvice = []
+  private final Set<Advice> consumerAdvice = [
+    Advice.ofRemove(projectCoordinates(':producer'), 'implementation')
+  ]
```

Comment:
Oo, turns out I already kind of had a test that showcased the bug, but didn't know it 😅